pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -1,22 +1,21 @@
 { stdenv, buildPackages, hostPlatform, fetchgit, perl, buildLinux, ... } @ args:
 
 buildLinux (args // rec {
-  version = "4.15.2018.04.14";
+  version = "4.15.2018.08.03";
   modDirVersion = "4.15.0";
```

Comment:
Looking at the `buildLinux` expression in https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/linux/kernel/generic.nix#L94, it adds bison for 4.16er kernels. Is it the case that the kernel version was rebased to a 4.16er kernel?

Reply:
yes indeed https://evilpiepirate.org/git/bcachefs.git/commit/?id=0adb32858b0bddf4ada5f364a84ed60b196dbcda

Reply:
How strange. I guess he didn't update the tag.